README.md

```diff
@@ -64,7 +64,18 @@ when you run `mdbook build`.
 
 10. Fix those links in the rustc-guide (by making a PR in the rustc-guide repo)
 
-11. Make a PR on the rust-lang/rust repo to update the rustc-guide git submodule in src/docs/rustc-guide
+11. Make a PR on the rust-lang/rust repo to update the rustc-guide git submodule in src/docs/rustc-guide.
+To make a PR, the following steps are useful.
+
+```bash
+# Assuming you already cloned the rust-lang/rust repo and you're in the correct directory
+git submodule update --remote src/doc/rustc-guide
+cargo update -p rustc-guide
```

Comment:
This step is not needed. I don't actually know what it does.

Reply:
Removed